examples/examples_nodejs_test.go

```diff
@@ -55,6 +55,18 @@ func TestWebserverTs(t *testing.T) {
 	integration.ProgramTest(t, &test)
 }
 
+func TestStorageTransferTs(t *testing.T) {
+	test := getJSBaseOptions(t).
+		With(integration.ProgramTestOptions{
+			Dir:         filepath.Join(getCwd(t), "storagetransfer-ts"),
+			SkipRefresh: true,
+		})
+
+	integration.ProgramTest(t, &test)
+	test.SkipRefresh = false
+	integration.ProgramTest(t, &test)
```

Comment:
Consider adding a comment about why we do it like this.

Reply:
I was doing this wrong. I fixed it now. I forgot we do an empty refresh anyway by default.